conda/recipes/dask-cuml/meta.yaml

```diff
@@ -28,29 +28,17 @@ build:
 
 requirements:
   build:
-    - python
-    - cudf
-    - dask >=0.19.0
-    - distributed >=1.23.0
     - dask-cudf
     - dask-cuda
   run:
-    - python
-    - cudf
-    - dask >=0.19.0
-    - distributed >=1.23.0
     - dask-cudf
     - dask-cuda
 test:
   imports:
-    - cudf
-    - dask
-    - distributed
-    - dask_cudf
-    - dask_cuda
```

Comment:
I'm not the most famliar with conda recipes so it would be good to get @dantegd's review as well. 

`LocalCUDACluster` is used in our tests, which comes from `dask_cuda`. Should that dependency be included?

Reply:
dask-cuda is kept as a dependency for dask-cuml in `run:` section also. `import dask_cuda` test is removed from `test:` section because it is anyway a dependency at runtime. `import dask_cuml` in `test:` section covers all these.